src/main/webapp/help/inheritFrom.html

```diff
@@ -7,4 +7,6 @@
 
 Volume inheritance works exactly as Container templates.
 
-Image Pull Secrets** are combined (all secrets defined both on 'parent' and 'current' template are used).
\ No newline at end of file
+Image Pull Secrets** are combined (all secrets defined both on 'parent' and 'current' template are used).
+
+Leaving this value empty will cause the podTemplate to inherit from a default build agent template.
```

Comment:
This needs to be reworded to indicate the following:
* nested podTemplate blocks inherits each other by default
* <default> means default inheritance is used
* '' means no inheritance

```suggestion
By default, pod template inherits outer pod template definitions. 
Inheritance can be stopped by using the empty string.
```